coss/discourse/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+from wagtail.wagtailadmin.edit_handlers import FieldPanel
+from wagtail.wagtailsnippets.models import register_snippet
+
+
+@register_snippet
+class DiscourseCategory(models.Model):
+    category_id = models.PositiveSmallIntegerField()
+    name = models.CharField(max_length=50, null=True)
+
+    panels = [
+        FieldPanel('category_id')
```

Comment:
Since we'll be fetching this from the API, I don't think we should make it user editable